### PR TITLE
Upgrade TensorFlow 2.1

### LINKED ISSUE #1010: Update Tensorflow versions
Tensorflow documentation seems to be stuck at version 1.8 which is > 1 year old now.
Since then there have been all the minors up to version 1.13 now. There is also the new version 2.0-alpha.

Each of those have many breaking changes and it's hard to work with conflicting dependency and documentations. Is it possible to add the missing versions, like it's done for other libraries like Numpy?

### LINKED ISSUE #1141: Tensorflow version are missing
How to automate the scrapping for tensorflow verisons? 
v 1.15
v 2.0  

Tensorflow API changes very fast, so this is useful.

### PR DESCRIPTION
**Pull request by lgeiger**
This PR upgrades TensorFlow documentation to version 2.1.

I also removed version 1.8 since it is not available on the website anymore. After this PR one could easily add support for version 2.0, 1.15 - 1.10.

Closes #1141
Closes #1010

## PR Checklist

- [x] Updated the versions and releases in the scraper file
- [x] Ensured the license is up-to-date and that the documentation's entry in the array in `about_tmpl.coffee` matches it's data in `self.attribution`
- [x] Ensured the icons and the `SOURCE` file in <code>public/icons/*your_scraper_name*/</code> are up-to-date if the documentation has a custom icon
- [x] Ensured `self.links` contains up-to-date urls if `self.links` is defined
- [x] Tested the changes locally to ensure:
  - The scraper still works without errors
  - The scraped documentation still looks consistent with the rest of DevDocs
  - The categorization of entries is still good

### CODE REVIEW

File: lib/docs/filters/tensorflow/entries.rb [hunk 1]
```diff
@@ -7,35 +7,108 @@ def get_name
...
+          'tf.tpu'
+        elsif slug.start_with?('tf/train')
+          'tf.train'
+        elsif slug.start_with?('tf/version')
+          'tf.version'
+        elsif slug.start_with?('tf/xla')
+          'tf.xla'
         else
-          node = at_css('.devsite-nav-item.devsite-nav-active')
-          node = node.ancestors('.devsite-nav-item').first.at_css('.devsite-nav-title')
```
Comment (lgeiger) [inline lib/docs/filters/tensorflow/entries.rb]: Unfortunately this breaks with the new version of the documentation, so I hard coded the top level navigation to be consistent with the TensorFlow website.

**Comment by Violet-Bora-Lee** [comment]
Could you update the status of Trello?

https://trello.com/b/6BmTulfx/devdocs-documentation

**Comment by lgeiger** [comment]
> Could you update the status of Trello?

@Violet-Bora-Lee Could you elaborate on what I should update in the Trello Board? I can't see any mention of it in the  [contributing guide](https://github.com/freeCodeCamp/devdocs/blob/master/.github/CONTRIBUTING.md#updating-existing-documentations).

**Comment by Violet-Bora-Lee** [comment]
It's not mention on the guide but i think it would be better to update the status on trello to share the status among contributors :) i was trying to start tf.js dosc but i found your PR.

**Comment by lgeiger** [comment]
:+1: I updated the [trello board](https://trello.com/c/1G7BTvQW/292-tensorflow)

**Comment by lgeiger** [comment]
@j-f1 Could you take a look this?

**Comment by j-f1** [comment]
Thank you for contributing!

**Comment by j-f1** [comment]
Small fixes in https://github.com/freeCodeCamp/devdocs/commit/bd1644a058e3ee67d83ac97825e9bcf4b0d69206

**Pull request merged**